### ISSUE TITLE
Litter Robot 4...

### ISSUE THREAD

**Issue by mschwartz** [natekspencer/pylitterbot#107]
reset_waste_drawer() throws "not implemented."  Ideally we could do an actual reset of the device, as in hitting the reset button.  But the Whisker app doesn't have this feature, so probably can't be done! :)

In the Whisker app, it shows:
Globe Light (on/off/auto)
Globe Brightness (low, medium, high)
Panel Brightness (low, medium, high)

It doesn't appear that pylitterbot supports the globe settings.  Maybe these are the nightlight ones?  Digging into the code, I thought maybe the night_light_level and night_light_brightness might be for the panel and globe brightness levels, but they are always the same value after changing night_light_brightness via the API calls.

I'm thinking the LR4 has a few buttons and lights on the top of the bonnet and the panel brightness would control the brightness of those top of the bonnet status lights?  And the glob light/brightness are what you call night_light.

<img width="1133" alt="image" src="https://user-images.githubusercontent.com/329637/213321315-fd2f8b81-9225-470b-8d10-cceac6c9a596.png">

**Comment by natekspencer** [comment]
> reset_waste_drawer() throws "not implemented."

`reset_waste_drawer` is only supported on the LR3. I need to do some cleanup and move that function to the LR3 module only, but it was there for backwards compatibility when I added the LR4 and changed a bunch of things around to reduce the headache of updating (though I'm not sure how much this module is used outside of Home Assistant)

> Ideally we could do an actual reset of the device, as in hitting the reset button. But the Whisker app doesn't have this feature, so probably can't be done! :)

The reset button on the robots seems to be a hardware only thing. I did try iterating through a number of different commands with the LR3 to see what was allowed, but never found one for this, so it can't be done currently.

> Globe Light (on/off/auto)
> Globe Brightness (low, medium, high)

These were renamed via later firmware and app updates, but "night light" was the original setting name and still controls the settings for the globe light/brightness. Night light level is just a helper to get/set the brightness level to a predefined name (low/medium/high) to match the app functionality. The LR4 actually supports a wider range of values here as well which is why there are two separate values: brightness (0..100) or level (low, medium, high). The brightness scale used to be 255, but was switched to 100 via a firmware update, haha.

> Panel Brightness (low, medium, high)

This was added after a firmware/app update and I haven't tackled it yet.
